src/components/dev-hub/star-rating.js

```diff
@@ -0,0 +1,180 @@
+import React, { useCallback, useReducer } from 'react';
+import PropTypes from 'prop-types';
+import StarIcon from '~components/dev-hub/icons/star-icon';
+import styled from '@emotion/styled';
+import { size, fontSize, screenSize } from './theme';
+
+const StyledContainer = styled.div`
```

Comment:
```suggestion
const StyledContainer = styled('div')`
```

Let's keep consistent by using `()` notation for these